pkg/smb/nodeserver_test.go

```diff
@@ -286,7 +283,6 @@ func TestNodePublishVolume(t *testing.T) {
 }
 
 func TestNodeUnpublishVolume(t *testing.T) {
-	skipIfTestingOnWindows(t)
 	errorTarget := "./error_is_likely_target"
```

Comment:
you need to fix the path, it's now hardcoded as Linux style

Reply:
use temp dir like this:
```
	existingMountPath, err := ioutil.TempDir(os.TempDir(), "csi-mount-test")
	if err != nil {
		t.Fatalf("failed to create tmp dir: %v", err)
	}
```

Reply:
Oops, got it. Misunderstood the original issue :)